packages/api-client/lib/index.ts

```diff
@@ -12,6 +12,7 @@ export type Topic =
   | 'ingestor_health'
   | 'fleet_states'
   | 'robot_health'
+  | 'task_summaries'
```

Comment:
Do you think that we can use an Emun here? I think it'd be ideal, but knowing about the type checking downside I'm not so sure. =|

Reply:
Mmm.. but it'll be easier for the user that imports that type and see the options as it is now.

Reply:
We need string literal types because we need to be compatible with the types of event emitter, not sure if enum can work.